kani-compiler/src/codegen_cprover_gotoc/codegen/operand.rs

```diff
@@ -484,13 +484,12 @@ impl<'tcx> GotocCtx<'tcx> {
         mem_place.address_of()
     }
 
-    /// Insert an allocation into the goto symbol table, and generate a goto function that will
-    /// initialize it.
+    /// Insert an allocation into the goto symbol table, and generate an init value.
```

Comment:
What does _allocation_ mean in this context ?

Reply:
It means a memory allocation.